include/utilities.h

```diff
@@ -36,20 +34,30 @@ std::string demangle(const char *name);
 // ======================================================================
 
 /**
- * Return a string containing the content of the vector, with elements
- * separated by the @ sep parameter.
+ * A compare function needed from the unique function.
  */
-template<class Type>
-std::string print(const std::vector<Type> &list, const std::string sep=",")
```

Comment:
because i made my usual mess with git... promise this is the last messy pull request.